htdocs/compta/facture/card.php

```diff
@@ -2782,7 +2820,8 @@
 	print '<td colspan="2">';
 	include_once DOL_DOCUMENT_ROOT . '/core/modules/facture/modules_facture.php';
 	$liste = ModelePDFFactures::liste_modeles($db);
-	print $form->selectarray('model', $liste, $conf->global->FACTURE_ADDON_PDF);
+	$curent = !empty($conf->global->{'FACTURE_ADDON_PDF_'.$object->type})?$conf->global->{'FACTURE_ADDON_PDF_'.$object->type}:$conf->global->FACTURE_ADDON_PDF;
```

Comment:
$conf->global->{'FACTURE_ADDON_PDF_'.$object->type}
seems not compatible with some php version. Can you replace with this way of coding ?

$paramkey='FACTURE_ADDON_PDF_'.$object->type
$conf->global->$paramkey